pom.xml

```diff
@@ -22,7 +22,7 @@ limitations under the License. -->
 
   <name>BlazingMQ Java SDK Bundle</name>
   <description>BlazingMQ Java SDK Bundle</description>
-  <url>https://github.com/bloomberg/blazingmq-sdk-java</url>
```

Comment:
maven flatten plugin just adds `bmq-sdk` or `bmq-examples` here in the end, and the link is not valid. `/tree/main/` allows to build these URLs correctly